test/CompareComply.v1.IntegrationTests/CompareComplyServiceIntegrationTests.cs

```diff
@@ -1,5 +1,5 @@
 ﻿/**
-* (C) Copyright IBM Corp. 2017, 2019.
+* (C) Copyright IBM Corp. 2017, 2020.
```

Comment:
I thought CnC came out in 2018.

Reply:
Hmm interesting. @mkistler it looks like this file was [created](https://github.com/watson-developer-cloud/dotnet-standard-sdk/commits/de1240d0fe63c404d373ab7331775fbf13de63ed/test/CompareComply.v1.IntegrationTests/CompareComplyServiceIntegrationTests.cs) in 2019 but `lint-copyright-dates` set the first date at 2017.